app/utils/MCLaunchCommand.js

```diff
@@ -73,11 +78,13 @@ const getStartCommand = async (packName, userData, settings, javaArguments) => {
     await promisify(fs.readFile)(path.join(PACKS_PATH, packName, 'config.json'))
   );
 
+  // TODO -- Change how -Djava.library.path is being used so it works the same as -cp.
+  // Store all natives in folder struc and just call it so it's no longer required in the instance folders.
```

Comment:
Does this still need to be done?

Reply:
No it does not.
It would be nice to have but i think the command line limit used to start would end up being to long at least for windows to work.